### PR TITLE
Handle call api list user error

### PR DESCRIPTION
**Pull request by AzizOuertani**
- added an error message if list of users loading fails .

-  added a refresh button to refresh the page.

image

### CODE REVIEW

File: src/locales/fr/users.json [hunk 1]
```diff
@@ -23,12 +23,17 @@
...
+      "title": "Il semblerait qu’il y est un problème. ",
+      "description": "Échec du chargement des utilisateurs"
     }
   },
   "list": {
     "title": "Gestion des utilisateurs",
     "actions": {
-      "createUser": "Créer un utilisateur"
+      "createUser": "Créer un utilisateur",
+      "ReloadPage": "Recharger la page"
```
Comment (Jessy-BAER) [inline src/locales/fr/users.json:36]: The key does not start with uppercase

File: src/app/admin/users/PageUsers.tsx [hunk 2]
```diff
@@ -256,93 +259,127 @@ export const PageUsers = () => {
...
-                    maxW="full"
-                    pointerEvents="auto"
-                    color="gray.600"
-                    _dark={{ color: 'gray.300' }}
+                <Box m="auto">
+                  <Button
+                    variant="@secondary"
+                    size="sm"
+                    leftIcon={<FiRefreshCw />}
+                    onClick={() => window.location.reload()}
```
[the remark this answers is not in the document]
Comment (ivan-dalmet) [reply src/app/admin/users/PageUsers.tsx]: In this case we don't want the whole page to reload but just refetch the query from react-query.
Don't use window.location.reload() here ;)

**Review by Jessy-BAER — APPROVED** [review]

**Review by ivan-dalmet — CHANGES_REQUESTED** [review]
The normal loading behavior is broken 😉
If you refocus the window or on page changed, we should not see a hard loading state

In this case, `isLoadingPage` should not be used to display or not the datalist 😉

https://user-images.githubusercontent.com/9749061/180200079-a3846091-0542-442d-973c-3d19ecc5d00e.mov

**Comment by ivan-dalmet** [comment]
Also @AzizOuertani can we have a little video demo of the error behavior? (because we can't get an error in the preview env)

**Comment by AzizOuertani** [comment]
> Also @AzizOuertani can we have a little video demo of the error behavior? (because we can't get an error in the preview env)


https://user-images.githubusercontent.com/109508360/180217630-8930aa41-4e7e-4021-a326-dd44d51ab04b.mp4

**Comment by Jessy-BAER** [comment]
Maybe we must fix the display in footer, "Showing 1 to 0 of 0 results" seems to be not good ? @ivan-dalmet  are you ok with this ?

**Comment by ivan-dalmet** [comment]
> Maybe we must fix the display in footer, "Showing 1 to 0 of 0 results" seems to be not good ? @ivan-dalmet  are you ok with this ? 

Yes but it can be in another PR

File: src/app/admin/users/PageUsers.tsx [hunk 3]
```diff
@@ -256,6 +259,34 @@ export const PageUsers = () => {
             </DataListCell>
             <DataListCell colName="actions" colWidth="4rem" align="flex-end" />
           </DataListHeader>
+          {isError && (
+            <DataListRow>
+              <DataListCell gap={4}>
```
Comment (Malek-khannoussi) [inline src/app/admin/users/PageUsers.tsx:264]: remove the `m="auto"` from the childrens and `justifyContent="center"`

File: src/app/admin/users/PageUsers.tsx [hunk 4]
```diff
@@ -256,6 +259,34 @@ export const PageUsers = () => {
...
+                </Box>
+                <Text
+                  m="auto"
+                  color="gray.500"
+                  fontSize="lg"
+                  fontWeight="medium"
+                >
+                  {t('users:feedbacks.loadingUserError.title')}
+                </Text>
+                <Box m="auto">
```
Comment (Malek-khannoussi) [inline src/app/admin/users/PageUsers.tsx:276]: remove `Box` you dont need it

**Review by ivan-dalmet — APPROVED** [review]

**Pull request merged**